src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const { type, extraData, keysToRemove } of actions) {
```

Comment:
```suggestion
  for (action of actions) {
  const { 
    type, 
    extraData, 
    keysToRemove,
 } = action
```
Your code should be not just short, but also understandable and descriptive